README.md

```diff
@@ -120,6 +120,46 @@ The message will look like the following one:
         "thread": "main",
         "@version": "1"
     }
+    
+#### Renaming fields, exceptionFields, locationFields
+
+Existing field that needs to be renamed can be specified in the with `renamedFieldLabels`, `renamedExceptionFieldLabels` or `renamedLocationFieldLabels` configuration properties of the layout:
+    
+    log4j.rootLogger = INFO, stdout
+
+    log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+    log4j.appender.stdout.layout=org.jetbrains.appenders.JsonLayout
+    log4j.appender.stdout.renamedFieldLabels=level:gns-level,@version:@gns-version
+    log4j.appender.stdout.renamedExceptionFieldLabels=class:gns-level,@version:@gns-class
+    log4j.appender.stdout.renamedLocationFieldLabels=file:gns-file
+        
+The message will look like the following one:
```

Comment:
do we have the whole message as one-line. It is necessary for some logging systems. Probably it is worth to mention in the text that we show a formatted version of the JSON. 
Also new-lines in exception are still there, you may probably shorten the exception in the example

Reply:
The message will remain multiline. We actually need that for out logging system :)